bot/README.md

```diff
@@ -0,0 +1,63 @@
+# Code Coverage Bot
+
+This project runs as [Taskcluster hooks](https://firefox-ci-tc.services.mozilla.com/hooks) on the firefox-ci instance, to extract and store the code coverage information from mozilla-central and try builds.
+
+It's built using Python 3.8 and few dependencies.
+
+## Developer setup
+
+Requirements:
+
+- Python 3.8
+- [virtualenv](https://virtualenv.pypa.io/en/stable/)
+- [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
```

Comment:
In theory these are not required-required, you could run the project without them

Reply:
Added an optional tag + comment to reflect this